x/auth/ante.go

```diff
@@ -36,6 +38,20 @@ func NewAnteHandler(am AccountMapper, fck FeeCollectionKeeper) sdk.AnteHandler {
 				true
 		}
 
+		memo := tx.GetMemo()
+
+		if len(memo) > maxMemoCharacters {
+			return ctx,
+				sdk.ErrMemoTooLarge(fmt.Sprintf("maximum %d characters but was %d characters", maxMemoCharacters, len(memo))).Result(),
```

Comment:
Consider changing to something more verbose, e.g. "maximum number of characters is %d, but received %d characters"

Reply:
Updated.